src/pybind/mgr/dashboard/frontend/src/app/shared/directives/permission-scope.directive.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { PermissionScopeDirective } from './permission-scope.directive';
+import { AuthStorageService } from '../services/auth-storage.service';
+
+describe('PermissionScopeDirective', () => {
```

Comment:
Could you please add a test here that tests the behavior of the service?

Reply:
Yes. I will add the tests.